server/uno.go

```diff
@@ -280,6 +240,47 @@ func dealCards(game *model.Game) (*model.Game, error) {
 	return game, err
 }
 
+func checkForCardInHand(card model.Card, hand []model.Card) bool {
```

Comment:
Did you want to mark this as a section for utility functions? or just have them at the bottom of the file. I'm really ok either way. I just wanted to see which way you wanted it

Reply:
Lets mark it as utility functions

Reply:
They were just scattered through the file and i wanted to group them together.

Reply:
k. let me know when you've pushed up that section heading.

Reply:
Seaction header added to utility functions